app/routes/ready-reckoner-v2/sfi-summary.js

```diff
@@ -159,27 +164,27 @@ module.exports = [
           categoryAmounts[id][feature] = landValues[feature]
           landFeatures[feature].standards.forEach(standard => {
             if (selectedStandards.includes(standard)) {
+              console.log(paymentAmounts.standards[standard])
               categoryAmounts[id].visible = true
-              categoryAmounts[id].payment += paymentAmounts[standard].base
+              categoryAmounts[id].payment += paymentAmounts.standards[standard].base
             }
 
             standards.standards[standard].optionalActions.forEach(
-              action => (categoryAmounts[id].paymentOptional += paymentAmounts[standard].optional[action])
+              action => (categoryAmounts[id].paymentOptional += paymentAmounts.standards[standard].optional[action])
             )
           })
         })
 
         categoryAmounts[id].payment += categoryAmounts[id].paymentOptional
       })
 
-      return h.view(pageDetails.template, pageContent(categoryAmounts, actionValues, paymentAmounts))
-    }
-  },
-  {
-    method: 'POST',
```

Comment:
Removing the `POST` route breaks hitting the continue button on this page, so I'd put it back in.

Reply:
Ha! I thought it weird that it existed. You can tell I never proceed 🛑